content/en/agent/guide/why-should-i-install-the-agent-on-my-cloud-instances.md

```diff
@@ -10,20 +10,35 @@ further_reading:
+* **Better resolution** - Cloud providers monitor your hosts externally by sampling them every 5-25 minutes. Additionally, AWS provides metrics on a per minute basis through their API. As Datadog stores all metrics at a 1-second resolution, AWS metrics are averaged over 60 seconds during post-processing. To provide more granular insight into host performance, the Datadog Agent collects performance statistics every 15 seconds, offering a more detailed view of what's happening inside your hosts.
+
+  {{< img src="agent/guide/Agent_VS_AWSA.jpg" alt="Agent vs AWS CloudWatch" style="width:70%;">}}
 
-* **Exposed metrics** - Datadog has over 50 metrics enabled by default. More metrics can be added with Datadog's application-specific integrations.
+* **Exposed metrics** - Datadog has over 50 metrics enabled by default. More metrics can be added with Datadog's application-specific [integrations][2].
 
-* **Integrations** - These make it simple to extend the Datadog Agent beyond the native metrics so you can monitor application health, process utilization, and more.
+* **Integrations** - Over [750+ integrations][2] extend the functionality of the Datadog Agent beyond the native metrics.
 
-* **Custom metrics with DogStatsD** - With the Datadog Agent on board, use the built-in StatsD client to send custom metrics from your application, allowing you to correlate what's happening with your application, your users, and your system.
+* **Tagging Consistency Across Products**: Tags applied at the Agent-level are applied to all metrics, logs, traces, etc. reported by the Agent. 
 
-  {{< img src="agent/guide/Agent_VS_AWSA.jpg" alt="Agent vs AWS CloudWatch" style="width:70%;">}}
+* **Custom metrics with DogStatsD** - With the Datadog Agent, use the built-in [StatsD client][4] to send custom metrics from your application, allowing you to correlate what's happening with your application, your users, and your system.
 
-The Datadog Agent is lightweight and fully open source, so you can review the code and even contribute by making a pull request.
+* **Custom Agent Checks** - For even deeper customization, implement [custom Agent checks][5] to collect metrics and other data from your custom systems or applications and send them to Datadog.
+
+* **Application Logs**: The Datadog Agent [collects and forwarder application logs that are created locally][6] on your cloud VMs or containers so they don't need to be forwarded through the Cloud-provider. These logs also have Agent-level tags applied.
+
+* **Application Performance Monitoring (APM)** - [Traces collected through the APM Agent][4] give a comprehensive look into your applications, helping you understand end-to-end service performance and identify potential bottlenecks.  
```

Comment:
```suggestion
* **Application Performance Monitoring (APM)** - [Traces collected through the Agent][4] give a comprehensive look into your applications, helping you understand end-to-end service performance and identify potential bottlenecks.  
```

Reply:
I know that `APM Agent` is referenced elsewhere in the docs, but it's not on the page being linked here, and I can imagine this leading to confusion about whether the `APM Agent` is separate from the main Agent. Alternatively, updating to `APM Agent process` might work as well.